### PR TITLE
add create wishlist item and get an item for the return statement

### LINKED ISSUE #8: Create a wishlist item
**User Story:**
As a user, I want to be able to add a product to my product wishlist, so that I can revisit the item(s) at a later time if necessary.

**Assumptions:**
* Wishlist CRUD functionality exists

**Acceptance Criteria:**
AC1) I can add an item to a wishlist
AC2) Endpoint should be ```/wishlist/:WishlistId/item/:ItemId```
AC3) Endpoint should accept ```POST``` method
AC4) Endpoint should return HTTP 200 on success
AC5) Endpoint should return HTTP 400 if ```:WishlistId``` or ```:ItemId``` is invalid
AC6) Endpoint should return HTTP 500 for server errors

**Testing Plan:**
* test should confirm correct http responses and codes for AC4, AC5 and AC6
* test should confirm database record exists on success

### PR DESCRIPTION
**Pull request by ZiyiHuang96**
run nosetests to verify that work

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "DATABASE_URI", "postgres://postgres:postgres@localhost:5432/testdb"
 )
 BASE_URL = "/wishlists"
+ITEM_URL = BASE_URL + "/1/items"
```
Comment (billykern) [inline tests/test_routes.py:14]: should we assume it is always `1` or take this value from whatever the factory generates?

**Review by billykern — APPROVED** [review]
one comment, but please merge after other PR with `get_items` gets merged

File: tests/test_routes.py (hunk 1 above)
Comment (biegel) [reply tests/test_routes.py:14]: hmmm... good point.  does `nose` always wipe the database clean before each run?

Comment (ZiyiHuang96) [reply tests/test_routes.py:14]: It will wipe the db each time, this is fine but I can change it.

Comment (biegel) [reply tests/test_routes.py:14]: let's leave it 👍

**Review by biegel — APPROVED** [review]

**Pull request merged**